src/client/mtna/archive-template.jade

```diff
@@ -19,6 +19,7 @@ div.relative(role="main"
     md-card(
       ng-if="state.pre.length > 0"
       flex-gt-sm="90" flex-gt-md="80"
+      class="firefox-tweak"
```

Comment:
Prefer using `md-card.firefox-tweak` syntax for declaring classes.

Also, is this necessary for all md-card, or just for the record-viewer card?